### PR TITLE
Pushes wallet extension noise to logs.

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

The wallet extension is too loud by default, logging ~four lines for every request (there can be 1+ requests per second).

Ticket: https://github.com/obscuronet/obscuro-internal/issues/633.

### What changes were made as part of this PR:

Functional.

- Removes unnecessary logging of request encryption/response decryption
- Pushes logging to optional log file, with the exception of errors, which are always printed as well

### CODE REVIEW

File: tools/walletextension/main/cli.go [hunk 1]
```diff
@@ -20,17 +20,23 @@ const (
 	nodeRPCWebsocketAddressName    = "nodeRPCWebsocketAddress"
 	nodeRPCWebsocketAddressDefault = "testnet.obscu.ro:13001"
 	nodeRPCWebsocketAddressUsage   = "The address on which to connect to the node via RPC using websockets"
+
+	logPathName    = "logPath"
+	logPathDefault = ""
```
Comment (tudor-malene) [inline tools/walletextension/main/cli.go:25]: shouldn't it log by default in like a :"wallet-extension.log" file, in the current folder

**Review by tudor-malene — COMMENTED** [review]
one small comment

File: tools/walletextension/main/cli.go (hunk 1 above)
Comment (joeldudley) [reply tools/walletextension/main/cli.go]: Addressed. I chose `wallet_extension_logs.txt`, for consistency with the enclave and host logs.

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**